tests/js/common/shell/arangosearch-update-consistency.js

```diff
@@ -54,9 +76,169 @@ function UpdateConsistency() {
         c.update("279974", {value: 345, flag: false});
       }
     },
+
+    testSingleleUpdateUnlinkedField: function () {
+      let c = db._collection(collection);
+      c.save({_key: "279974", value: 345, unlinkedField: 456, flag: true});
+      db._createView(view, "arangosearch", {links: {testUpdate: {"fields": {"value": {"analyzers": ["identity"]}}}}});
+
+      for (let j = 0; j < 100; j++) {
+        let oldLen = db._query("FOR d IN " + view + " SEARCH d.value == 345 RETURN d").toArray().length;
+        assertEqual(oldLen, 1);
+        c.update("279974", {unlinkedField: j, flag: true});
+        let newLen = db._query("FOR d IN " + view + " SEARCH d.value == 345 RETURN d").toArray().length;
+        assertEqual(newLen, 1);
+        c.update("279974", {unlinkedField: j, flag: false});
+      }
+    },
+
+    testMultiUpdatesIncludeAllFields: function () {
+      let collectionProperties = {};
+      if (isCluster) {
+        collectionProperties = {numberOfShards: 9, replicationFactor: 3};
+      }
+
+      const amountOfCollections = 3;
+      const amountOfDocuments = 10;
+      const documents = [];
+
+      for (let i = 0; i < amountOfDocuments; i++) {
+        documents.push({_key: `doc${i}`, value: i});
+      }
+
+      v = db._createView(view, "arangosearch", {})
+      const insertedDocs = [];
+      for (let i = 0; i < amountOfCollections; i++) {
+        db._create(`testUpdate${i}`, collectionProperties);
+        insertedDocs.push(db[`testUpdate${i}`].insert(documents));
+        v.properties({links: {[`testUpdate${i}`]: {includeAllFields: true}}});
+      }
+
+      for (let j = 1; j < 100; j++) {
+        let oldLen = db._query("FOR d IN " + view + " SEARCH d.value >= 0 RETURN d").toArray().length;
+        assertEqual(oldLen, amountOfCollections * amountOfDocuments);
+        for (let i = 0; i < amountOfCollections; i++) {
+          for (let j = 0; j < amountOfDocuments; j++) {
+            db[`testUpdate${i}`].update(`doc${j}`, {value: j, flag: true});
+          }
+        }
+        let newLen = db._query("FOR d IN " + view + " SEARCH d.value >= 0 RETURN d").toArray().length;
+        assertEqual(newLen, amountOfCollections * amountOfDocuments);
+        for (let i = 0; i < amountOfCollections; i++) {
+          for (let j = 0; j < amountOfDocuments; j++) {
+            db[`testUpdate${i}`].update(`doc${j}`, {value: j, flag: false});
+          }
+        }
+      }
+
+      for (let i = 0; i < amountOfCollections; i++) {
+        db._drop(`testUpdate${i}`, collectionProperties);
+      }
+    }
+  };
+}
+
+function UpdateConsistencySearchAliasView() {
```

Comment:
search-alias view test and arangosearch view test different only in setup part (one needs to create inverted index + search-alias view another arangosearch view).
So please write single test with parameter